microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-examples.adoc

```diff
@@ -0,0 +1,119 @@
+= Microsoft .NET Connector 3.1 Examples - Mule 4
+
+The following use case...
```

Comment:
The following use case will execute method `SumOfDigits(System.Int32 number) -> System.Int32` from library `DemoDotNet.dll`